docs/en/04-supported-codebases/mmdet3d.md

```diff
@@ -79,5 +80,6 @@ The caller needs to refer to the corresponding [python implementation](../../../
 | [centerpoint](https://github.com/open-mmlab/mmdetection3d/blob/main/configs/centerpoint/centerpoint_pillar02_second_secfpn_head-circlenms_8xb4-cyclic-20e_nus-3d.py) | nuScenes |     ✔️      |    ✔️    |     ✔️     |
 |             [pointpillars](https://github.com/open-mmlab/mmdetection3d/blob/main/configs/pointpillars/pointpillars_hv_secfpn_sbn-all_8xb4-2x_nus-3d.py)              | nuScenes |     ✔️      |    ✔️    |     ✔️     |
 |            [pointpillars](https://github.com/open-mmlab/mmdetection3d/blob/main/configs/pointpillars/pointpillars_hv_secfpn_8xb6-160e_kitti-3d-3class.py)            |  KITTI   |     ✔️      |    ✔️    |     ✔️     |
+|                   [smoke](https://github.com/open-mmlab/mmdetection3d/blob/main/configs/smoke/smoke_dla34_dlaneck_gn-all_4xb8-6x_kitti-mono3d.py)                    |  KITTI   |     ✔️      |    ✔️    |     ✔️     |
```

Comment:
Could we add one more column `Task` to make it better ?

Reply:
OK.

Reply:
Done.